fronts-client/src/services/pandaFetch.ts

```diff
@@ -22,16 +22,16 @@ const pandaFetch = (
         credentials: 'same-origin',
       });
 
-      if (res.status === 419 && count < 1) {
+      if ((res.status === 419 || res.status === 401) && count < 1) {
```

Comment:
Our endpoints return `401` when authentication fails – I'm not sure why we used `419` here, [it's not an official HTTP status](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)

Reply:
Ah, got it – panda issues that status when oauth has expired. So we catch 401s here from the app itself, and 419s from panda.